### ISSUE TITLE
Cromedriver 115 not working

### ISSUE THREAD

**Issue by serdnago** [ElSnoMan/pyleniumio#313]
Aloha!,

This started to happen this week:     

   if seleniumwire_options is None:  # default to regular ChromeDriver
            driver = webdriver.Chrome(
>               service=ChromeService(local_path or ChromeDriverManager(version=version).install()),
                options=browser_options,
                **(webdriver_kwargs or {}),
            )
E           TypeError: ChromeDriverManager.__init__() got an unexpected keyword argument 'version'

**Comment by grodowski** [comment]
Starting a small project with `pyleniumio` and ran into the same problem. I think this is the change in `webdriver_manager` that causes it: https://github.com/SergeyPirogov/webdriver_manager/commit/06997dc017cf64c8ed5f8e735fe2f2d0af3e7f20#diff-38d3c158b7a31027012330b5537df014f3ec50ffa657a2787d449bd3e3c9367bL13 

I added `webdriver_manager = "^3.8.6"` as explicit dependency for now and set `version: "114.0.5735.90"` in `pylenium.json`. This does workaround the `unexpected keyword argument 'version'`, but there's another error when you leave a blank `version`:

```
ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/LATEST_RELEASE_115.0.5790
```

The latter is probably related to changes in SeleniumManager: https://stackoverflow.com/questions/76724939/there-is-no-such-driver-by-url-https-chromedriver-storage-googleapis-com-lates

If anyone has tips how to take this further, I might be able to invest some time in a fix.

**Comment by ElSnoMan** [comment]
We do have the option of moving away from `webdriver_manager` and just using `Selenium Manager` that's built into Selenium. I was waiting for it to be more stable, but it could be a good time to evaluate it since that's ultimately what I'd want to move to:

https://stackoverflow.com/questions/76724939/there-is-no-such-driver-by-url-https-chromedriver-storage-googleapis-com-lates

**Comment by 5SIMU** [comment]
Hello,
I have got this error:
"TypeError: ChromeDriverManager.__init__() got an unexpected keyword argument 'version'
..\venv\Lib\site-packages\pylenium\webdriver_factory.py:203: TypeError" 

having the latest version of webdriver_manager 4.0.0 and Chrome 115.0.5790.171.

"pylenium.json" file has "version": null, and "seleniumwire_enabled": false, . I tried also with "version": "", but the same error.

As a solution I did the following changes in webdriver_factory.py -> build_chrome():

    if seleniumwire_options is None:  # default to regular ChromeDriver
        driver = webdriver.Chrome(
            #service=ChromeService(local_path or ChromeDriverManager(version=version).install()),
            service=ChromeService(local_path or ChromeDriverManager().install()),
            options=browser_options,
            **(webdriver_kwargs or {}),
        )
This is working ,currently for me, but don't know if it will work for a long time.
I am really interested how to resolve this issue, because I just started to work with pylenium.

Thanks!

**Comment by IgBuS** [comment]
> We do have the option of moving away from `webdriver_manager` and just using `Selenium Manager` that's built into Selenium. I was waiting for it to be more stable, but it could be a good time to evaluate it since that's ultimately what I'd want to move to:
> 
> https://stackoverflow.com/questions/76724939/there-is-no-such-driver-by-url-https-chromedriver-storage-googleapis-com-lates

Hi! Thank you for your library, it is very helpful.
I faced the same problem it may be solved by using this version of  webdriver-manager = "==3.8.6"
maybe it is a good idea to fix it?

**Comment by IgBuS** [comment]
https://github.com/ElSnoMan/pyleniumio/pull/316
I hope this is what is needed

**Comment by LostInDarkMath** [comment]
https://github.com/ElSnoMan/pyleniumio/pull/310 would also fix this issue by implementing the API changes

**Comment by ElSnoMan** [comment]
This has been fixed in https://github.com/ElSnoMan/pyleniumio/pull/317 and will be released in `v1.20.0`
